guides/video_pagination.md

```diff
@@ -0,0 +1,138 @@
+
+You should call `startLocalVideo()` when you want to start sending local video to others, and call `stopLocalVideo()` when you want to stop sending local video to others (e.g. when the app is in background). Stopping local video will save uplink bandwidth and computation resource for encoding video.
+
+### Start/Stop Remote Video
+
+Similar to `startLocalVideo()` and `stopLocalVideo()`, `startRemoteVideo()` and `stopRemoteVideo()` will start and stop receiving remote attendees’ videos by setting the service type of video client and turning on/off its *receiving* mode.
+
+Note that when the application calls `stopRemoteVideo()`, all the resources in the render pipeline will be released, and it takes some time to re-initialize all these resources. You should call `stopRemoteVideo()` only when you want to stop receiving remote videos for a *considerable amount of time*. If you want to temporarily stop remote videos, consider iterating through remote videos and call `pauseRemoteVideoTile()` instead.
+
+### Pause/Resume Remote Video Tile
+
+You can call `pauseRemoteVideoTile()` to temporarily pause a remote video stream. When you call `pauseRemoteVideoTile()`, the video client will add the tile id to a local block list, so that server will not forward video frames from the paused stream to the client. In that case, you can save downlink bandwidth and computation resource for decoding video. You can call `resumeRemoteVideoTile()` to remove a video from the local block list and resume streaming.
+
+You should call `pauseRemoteVideoTile()` when you want to temporarily pause remote video (e.g. when user is not in the video view and remote videos are not actively being showed), or when you want to stop *some* remote videos (e.g. in the video pagination example in the following sections, where we want to render videos on the current page, but stop invisible videos on other pages).
+
+### Bind/Unbind Video View
+
+To display video, you need to bind a video tile to a video view. You can call `bindVideoView()` to bind a video tile to a view, and call `unbindVideoView()` to unbind a video tile from a view.
+
+Note that bind/unbind only happens on the *representation layer*, which means these APIs only control when and where the video stream will be displayed in your application UI. If you unbind a video tile, the server will keep sending video frames to the client, and the client will continue consuming resources to receive and decode video. If you want to reduce unnecessary data consumption and CPU usage of your application by stopping remote videos, you should call `pauseResumeVideo()` first.
+* * *
```

Comment:
what is this three asterisks?

Reply:
Three asterisks were rendered as a horizontal line.